include/ClientLogger.h

```diff
@@ -37,7 +37,7 @@ class ClientLogger {
 * @brief   Values that represent logging levels.
 */
     enum CLIENT_LOG_LEVEL {
-        ERROR    = 1,
+        ERR      = 1,
```

Comment:
It was not letting compile in Windows environment even though it's part of enum. Was complaining about the predefined ERROR in Windows environment, so changed name to ERR instead.